packages/mgt-components/src/utils/Utils.ts

```diff
@@ -47,6 +47,50 @@ export function getRelativeDisplayDate(date: Date): string {
   });
 }
 
+/**
+ * returns a promise that resolves after specified time
+ * @param time in milliseconds
+ */
+export function delay(ms: number): Promise<void> {
```

Comment:
This function exists in `@microsoft/mgt-element` and is exported for use, we shouldn't be duplicating code like this

Reply:
Should be cleaned up.